### DISCUSSION TITLE
Customizing repo server git config

### DISCUSSION THREAD

**Discussion by ramanNarasimhan77** [argoproj/argo-cd#6983] [Q&A] [answered]
Hello,
I am trying to deploy an application from a git repo by connecting over HTTP.

Often the git fetch commands fail with errors like `Remote hung up unexpectedly`

I want to try out some git configurations like increasing http post buffer size etc, but I am not sure if that is a good idea.

Also, I noticed that some of the Git operations in Argocd use the `go-git` SDK instead of the git cli. Does the `go-git` SDK also respect the git configuration if its specified in $HOME/.gitconfig ?

**Comment by jessesuen** [comment, marked as answer]
No I don't believe so. According to [go-git docs](https://github.com/go-git/go-git/blob/bf3471db54b0255ab5b159005069f37528a151b7/COMPATIBILITY.md#supported-capabilities), `$HOME/.gitconfig` is not respected at the system level, only at repo level.

**Reply by ramanNarasimhan77** [reply]
Thank you for the response @jessesuen
